CONTRIBUTING.md

```diff
@@ -15,28 +15,55 @@ No software is bug free. So, if you got an issue, follow these steps:
   * Share error logs, screenshots and etc.
   * To speed up the issue fixing process, send us a sample repo with the issue you faced:
 
+### Testing against `master`
+
+To test your project against the current latest version of storybook, you can clone the repository and link it with `npm`. Try following these steps:
+
+1. Download the latest version of this project, and build it
+
+  ```
+  git clone https://github.com/storybooks/storybook.git
+  cd storybook
+  npm install
```

Comment:
@ndelangen where are we at with fixing this problem? Should we update these instructions or will it be sorted soon?